usr/lib/systemd/system/greenboot-healthcheck.service

```diff
@@ -17,3 +17,4 @@ PrivateMounts=yes
 [Install]
 RequiredBy=boot-complete.target
 WantedBy=multi-user.target
+Also=greenboot-success.target
```

Comment:
I saw the lines removing `greenboot-rollback.service` and `greenboot-success.target` above but I only see `greenboot-success.target` being added back, is `greenboot-rollback.service` now defunct?

Reply:
Gotcha, thanks for the full rundown